features/adb-openshift.feature

```diff
@@ -30,6 +34,11 @@ Feature: Command output from various OpenShift related commands in ADB
     ==> default: OpenShift service configured successfully...
     """
 
+    When I successfully run `sleep 20`
+    When I successfully run `bundle exec vagrant service-manager status`
+    Then stdout from "bundle exec vagrant service-manager status" should contain "openshift - running"
+
+    When I successfully run `sleep 20`
```

Comment:
Also, I think we are better of to create our own step definition. Something like "When I sleep 20 seconds". Then in the implementation you are calling Ruby's sleep method. This way you are doing things within Ruby. Using `sleep 2-` effectively calls the shells sleep command which might or might not work on Windows.